components/brave_shields/browser/ad_block_engine.cc

```diff
@@ -254,8 +254,9 @@ void AdBlockEngine::UpdateAdBlockClient(
     const std::string& resources_json) {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
   ad_block_client_ = std::move(ad_block_client);
-  if (regex_discard_policy_)
+  if (regex_discard_policy_) {
     ad_block_client_->setupDiscardPolicy(*regex_discard_policy_);
+  }
```

Comment:
I have no idea why presubmit complains about this formatting, even I didn't touch this file and the previous version looks good for me. 🤷